weblate/trans/tests/test_selenium.py

```diff
@@ -63,10 +63,11 @@
 SOURCE_FONT = os.path.join(
     os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
     "static",
+    "js",
     "vendor",
+    "fonts",
     "font-source",
-    "TTF",
-    "SourceSans3-Bold.ttf",
+    "SourceSans3-Bold.ttf.woff2",
```

Comment:
I guess this is why the tests fails:

```suggestion
    "SourceSans3-Bold.ttf",
```

Reply:
I rerun the test with the original file (and without deleting the asset .ttf) but it seems not the reason

Reply:
woff2 upload is directly rejected, so the test with it cannot pass.

Reply:
> woff2 upload is directly rejected, so the test with it cannot pass.

The .ttf version isn't included in the bundle by webpack because it isn't included in the css we import. So .woff2 was used instead. 

Btw Why woff2 isn't supported?